src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,4 +17,8 @@
         <servlet-name>dispatcher</servlet-name>
         <url-pattern>/*</url-pattern>
     </servlet-mapping>
+
+    <display-name>dogg</display-name>
+    <absolute-ordering />
```

Comment:
Do you understand these options? Why do you need them?

Reply:
The **<display-name>** is the "external name" of the servlet - it's what should show up on a management consoles of servlet containers, whereas the <servlet-name> is the "internal name" and we use it to refer to the servlet in other tags.
Using **<absolute-ordering>** element in deployment descriptor (web.xml), we can set absolute ordering of _web-fragments_. An empty <absolute-ordering /> element configures that none are to be scanned, but why do we need to explicity specify it?

Reply:
> but why do we need to explicity specify it?

Do you know what a web-fragment is? I don't. I don't know why you added these at all. You should understand every line that you add - both what it means and why you're adding it.

Reply:
Hm, I thought you did it. Ok.